backend/src/main/java/com/htecgroup/SkyNest/exception/UserExceptionHandler.java

```diff
@@ -0,0 +1,27 @@
+package com.htecgroup.SkyNest.exception;
+
+import com.htecgroup.SkyNest.model.response.ErrorMessage;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+@ControllerAdvice
+public class UserExceptionHandler {
+
+  private static final DateTimeFormatter formatter =
```

Comment:
This is probably a good candidate for Util method/class.  Usually those kinds of variables are used in multiple places. You can create it to be more generic as well, for example to have a default option and one to propagate different pattern.

Reply:
Agreed, will move it to util.